c2corg_ui/static/js/utils.js

```diff
@@ -123,6 +123,30 @@ app.utils.formatDate = function(date) {
 
 
 /**
+ * check that the start date if before end date.
+ * Returns an error message if there is an error with dates.
+ * @export
+ * @param {Object} outing
+ * @return {(undefined | string)}
+ */
+app.utils.checkDates = function(outing) {
+  var start = new Date(outing['date_start']);
+  var end = new Date(outing['date_end']);
+
+  if (!end) {
+    return;
+  }
+  if (start > end) {
+    return 'The starting date is after the ending date.';
```

Comment:
I had the same problem recently but couldn't find the right way to do it. But it should be possible, see https://github.com/c2corg/v6_ui/blob/master/Makefile#L104-L105

To mark the string as extractable, one should use something like ``this.gettextCatalog_.getString('some i18n string')`` as in https://github.com/c2corg/v6_ui/commit/58ef63d8135cac5aa1bc3c9993ef34f8ff70901e but that's probably not the correct syntax.

@tsauerwein @gberaudo do you know the right way to do it?